test/microapps-ui-xcm/microapps-ui-main.liq.position.test.ts

```diff
@@ -0,0 +1,424 @@
+/*
+ *
+ * @group microappsPosition
+ */
+import { jest } from "@jest/globals";
+import { Keyring } from "@polkadot/api";
+import { WebDriver } from "selenium-webdriver";
+import { getApi, initApi } from "../../utils/api";
+import { DriverBuilder } from "../../utils/frontend/utils/Driver";
+import {
+  addExtraLogs,
+  importPolkadotExtension,
+} from "../../utils/frontend/utils/Helper";
+import { AssetWallet, User } from "../../utils/User";
+import { getEnvironmentRequiredVars } from "../../utils/utils";
+import {
+  KSM_ASSET_ID,
+  MGA_ASSET_ID,
+  TUR_ASSET_NAME,
+} from "../../utils/Constants";
+import { Node } from "../../utils/Framework/Node/Node";
+import "dotenv/config";
+import {
+  connectWallet,
+  setupPage,
+  setupPageWithState,
+  waitForMicroappsActionNotification,
+} from "../../utils/frontend/microapps-utils/Handlers";
+import { ApiContext } from "../../utils/Framework/XcmHelper";
+import XcmNetworks from "../../utils/Framework/XcmNetworks";
+import { connectVertical } from "@acala-network/chopsticks";
+import { AssetId } from "../../utils/ChainSpecs";
+import { BN_TEN, BN_THOUSAND } from "@mangata-finance/sdk";
+import { BN } from "@polkadot/util";
+import StashServiceMockSingleton from "../../utils/stashServiceMockSingleton";
+import { LiqPools } from "../../utils/frontend/microapps-pages/LiqPools";
+import { Sidebar } from "../../utils/frontend/microapps-pages/Sidebar";
+import {
+  KSM_ASSET_NAME,
+  MGX_ASSET_NAME,
+} from "../../utils/frontend/microapps-pages/UiConstant";
+import { LiqPoolDetils } from "../../utils/frontend/microapps-pages/LiqPoolDetails";
+//import { Polkadot } from "../../utils/frontend/pages/Polkadot";
+import { TransactionType } from "../../utils/frontend/microapps-pages/NotificationModal";
+import { PositionModal } from "../../utils/frontend/microapps-pages/PositionModal";
+
+jest.spyOn(console, "log").mockImplementation(jest.fn());
+
+jest.setTimeout(1500000);
+let driver: WebDriver;
+let testUser1: User;
+
+const acc_name = "acc_automation";
+const userAddress = "5FeYEhdFzSQP6YWapPC6Myd17zdPba9CXmSptNSTh4Hz9cZ9";
```

Comment:
Please use same user as in other tests:
5CfLmpjCJu41g3cpZVoiH7MSrSppgVVVC3xq23iy9dZrW2HR